### PR TITLE
Corrected action taken after successfully storing the fingerprint in enroll example.

### PR DESCRIPTION
**Pull request by miselico**
A return statement after successfully storing the fingerprint was missing. Added a return value of 0.
When the fingerprint is taken successfully, the program should restart instead of continuing to take the fingerprint; amended.
The program prints dots to indicate it is waiting, removed a newline being printed after each of these in one case.

### CODE REVIEW

**Comment by caternuson** [comment]
> A return statement after successfully storing the fingerprint was missing.
Not missing, it's the `return true` after the conditional block. That would then cause the `  while (!  getFingerprintEnroll() );` to exit and then the main `loop()` would restart.

>  The program prints dots to indicate it is waiting, removed a newline being printed after each of these in one case.
Yah, that's annoying. The second read loop doesn't do that.

If this PR is still being monitored, please provide more info about the motivation for the added `return`.

**Comment by miselico** [comment]
That `return true` was added in 31585c9dba017a5c344dcd1cedc5be18b6337af2 , half a year after my pull request...

**Comment by miselico** [comment]
Let me pull the latest to my branch,

**Comment by miselico** [comment]
I'd still like the return in the if block more, because it is the only case in which that return is used, but that's a matter of preference.

File: examples/enroll/enroll.ino [hunk 1]
```diff
@@ -84,8 +86,11 @@ void loop()                     // run over and over again
   }
   Serial.print("Enrolling ID #");
   Serial.println(id);
-
-  while (!  getFingerprintEnroll() );
+  //assume an error happened
+  int error = 1;
+  while (error){
+    error = getFingerprintEnroll();
+  }
```
Comment (caternuson) [inline examples/enroll/enroll.ino:93]: This is functionally equivalent. Why the change?

**Review by caternuson — CHANGES_REQUESTED** [review]

File: examples/enroll/enroll.ino (hunk 1 above)
Comment (miselico) [reply examples/enroll/enroll.ino:93]: It has been a long time. I think I changed it because it took me a long time to figure out what the original actually meant.

Comment (caternuson) [reply examples/enroll/enroll.ino:93]: OK, for now plz make the PR just the fix for the "." prints.

Comment (miselico) [reply examples/enroll/enroll.ino]: Done.

**Comment by caternuson** [comment]
Thanks. LGTM.
